deploy/Gruntfile.js

```diff
@@ -1,6 +1,5 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
-const path = require('path');
```

Comment:
commenting to just making sure that this is intentional.

Reply:
yes, `path` is not used anywhere on the file